
public class Day6Demos {

	
	
	public static void main(String[] args) {
	
		int []x= {1,0};
		int y[]=null;
		System.out.println("lines b4 exception");
		
		System.out.println("line 1");
		System.out.println("line 2");
		try {
		//System.out.println(10/0);
		//System.out.println(10/x[10]);
			
			System.out.println(10/0);
			
		}
		finally {
			System.out.println("finally block");
		}
		System.out.println("line after exception");
		
	}

}

// exception : abnormal condition occurs during program execution
//and terminates the flow of program

//exception handling : protects d program frm being terminated
//try , catch , finally , throw , throws 






